fast64_internal/sm64/sm64_objects.py

```diff
@@ -1194,7 +1218,22 @@ def drawWarpNodeProperty(layout, warpNode, index):
 		if warpNode.warpType == 'Instant':
 			prop_split(box, warpNode, 'warpID', 'Warp ID')
 			prop_split(box, warpNode, 'destArea', 'Destination Area')
-			prop_split(box, warpNode, 'instantOffset', 'Offset')
+			prop_split(box, warpNode, 'useOffsetObjects', 'Use Offset Objects?')
+			if warpNode.useOffsetObjects:
+				prop_split(box, warpNode, 'instantWarpObject1', 'Warp Start')
+				prop_split(box, warpNode, 'instantWarpObject2', 'Warp End')
+				writeBox = box.box()
+				if warpNode.instantWarpObject1 is None or warpNode.instantWarpObject2 is None:
+					writeBox.label(text='Both Objects must be selected for offset')
+				else:
+					writeBox.label(text='Current Offset: ')
+					difference = warpNode.calc_offsets_from_objects()
+					
+					writeBox.label(text='X: ' + str(int(difference.x)))
+					writeBox.label(text='Y: ' + str(int(difference.y)))
+					writeBox.label(text='Z: ' + str(int(difference.z)))
```

Comment:
non blocking suggestion, this is just my personal preference to use f strings instead of concatenation, though you can skip the cast to `int` here since you're doing that in `calc_offsets_from_objects`

```suggestion
					writeBox.label(text=f"X: {difference.x}")
					writeBox.label(text=f"Y: {difference.y}")
					writeBox.label(text=f"Z: {difference.z}")
```